backend/docker-compose.yml

```diff
@@ -4,9 +4,19 @@ services:
     build: .
     ports: 
       - "3001:3001"
-    # depends_on:
-    # links:
-    #  - mongo 
+    links:
+      - db:mysql
+    depends_on:
+      - db
     volumes:
       - ./:/usr/src/app
-    # command: [ "npm", "run"]
\ No newline at end of file
+
+  db:
+    image: mysql
+    command: --default-authentication-plugin=mysql_native_password
+    container_name: "mysql"
+    ports:
+      - "3306:3306"
+    environment: 
+      MYSQL_ROOT_PASSWORD: p@ssw0rd
```

Comment:
please use .env and get the value instead of hardcoding

Reply:
All .env stuff fixed